src/tests/ftest/io/dfuse_sparse_file.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+#import subprocess
+from getpass import getuser
+import paramiko
+
+from general_utils import get_remote_file_size
+#from ClusterShell.NodeSet import NodeSet
+#from command_utils import CommandFailure
+from ior_test_base import IorTestBase
+
+
+class DfuseSparseFile(IorTestBase):
+    """Dfuse Sparse File base class
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a DfuseSparseFile object."""
+        super(DfuseSparseFile, self).__init__(*args, **kwargs)
+        self.space_before = None
+#        self.ret = None
+
+    def tearDown(self):
```

Comment:
I guess we can skip this because IorTestBase also looks to stop dfuse.

Reply:
Done